projects/application/filswan.md

```diff
@@ -0,0 +1,77 @@
+## Project Name Filswan
+![Filswan](https://github.com/nebulaai/swan/blob/main/doc/swan.png)
+
+## Category 
+application
+
+## Project Description
+Swan is a marketplace for Filecoin miners, clients post/bidding deals the online.
+
+The platform also provides powerful deal lifecycle management, fast file retrieval, Petabyte-Scale data migration, content network delivery, multi-data center caching, encryption, micro payment and edge computing services. Miners on swan platform can provide additional services according to their infrastructure and capabilities. 
+
+
+## Previews
+### Active Miner List
+![Miner List](https://github.com/nebulaai/swan/blob/main/doc/dashboard.png)
+### Market Deal Bids
+![Market Deal Bid](https://github.com/nebulaai/swan/blob/main/doc/bid.png)
+### Development API Keys
+![Development API Key](https://github.com/nebulaai/swan/blob/main/doc/api_key.png)
+
+## Target Audience
+Filecoin miners, Storage Clients
+
+## Rough estimated user base (if applicable)
+
+~100 signed miners, 2000+ deals proceesed 
+
+## Github repo
+https://github.com/nebulaai/swan
+
+## Website
+https://www.filswan.com
+
+## Docs
+https://github.com/nebulaai/swan
+
+## Team Info
+
+NBFS Canada (Nebula AI inc.) is a container-based cloud computing solution provider headquartered in Montreal, Canada. NBFS is working on different aspect of ecosystem projects included but not limited to Filecoin mining, mining pool development, edge computing. Nebula AI is also a Canada’s Centre of Excellence in Next Generation Networks project, https://www.cengn.ca/nebula-ai
+
+At present, NBFS Canada has 5 mining nodes within Tier 3 Datacenter providing stable and efficient storage solutions for different Filecoin ecosystem projects, e.g Starling, Pinata
+
+
+### Team Size  
+10+
+
+### Team members  
+
+Charles Cao
+
+Rex Miller
+
+Boqian Wang
+
+Chi Zhang
+
+Yiming Liang
+
+Baoyuan Sun
+
+Yi Xie
+
+Mike Li
+
+
+## How the community can engage
+
```

Comment:
To join the show case, could you please create a discussion in this repo too?

Reply:
addhttps://github.com/filecoin-project/community/discussions/69